### ISSUE TITLE
Mangrove Propagule causing crashes in 1.19 

### ISSUE THREAD

**Issue by TalEdd** [LordMZTE/quickreplant#5]
It keeps crashing and giving this error, whenever a propagule is placed on any block:

```
Description: Unexpected error

java.lang.IllegalArgumentException: Cannot get property class_2746{name=hanging, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{minecraft:mud}
	at Not Enough Crashes deobfuscated stack trace.(1.19+build.2)
	at net.minecraft.state.State.get(State:98)
	at net.minecraft.block.PropaguleBlock.isHanging(PropaguleBlock:146)
	at net.minecraft.block.PropaguleBlock.canPlaceAt(PropaguleBlock:85)
	at de.mzte.quickreplant.PlantCheckKt.isFittingPlantItem(PlantCheck.kt:28)
	at de.mzte.quickreplant.QuickreplantKt.onBlockUse(Quickreplant.kt:25)
	at net.fabricmc.fabric.api.event.player.UseBlockCallback.lambda$static$0(UseBlockCallback.java:41)
	at net.minecraft.client.network.ClientPlayerInteractionManager.handler$bka000$interactBlock(ClientPlayerInteractionManager:3681)
	at net.minecraft.client.network.ClientPlayerInteractionManager.interactBlock(ClientPlayerInteractionManager:327)
	at net.minecraft.client.MinecraftClient.doItemUse(MinecraftClient:1711)
	at net.minecraft.client.MinecraftClient.handleInputEvents(MinecraftClient:1964)
	at net.minecraft.client.MinecraftClient.tick(MinecraftClient:1791)
	at net.minecraft.client.MinecraftClient.render(MinecraftClient:1101)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:734)
	at fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:37)
	at net.minecraft.client.MinecraftClient.modify$doe000$atTheEndOfSecondCatchBeforePrintingCrashReport(MinecraftClient:17939)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:761)
	at fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:37)
	at net.minecraft.client.MinecraftClient.modify$doe000$atTheEndOfSecondCatchBeforePrintingCrashReport(MinecraftClient:17939)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:761)
	at net.minecraft.client.main.Main.main(Main:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_2688.method_11654(class_2688.java:98)
	at net.minecraft.class_7115.method_41435(class_7115.java:146)
	at net.minecraft.class_7115.method_9558(class_7115.java:85)
	at de.mzte.quickreplant.PlantCheckKt.isFittingPlantItem(PlantCheck.kt:28)
	at de.mzte.quickreplant.QuickreplantKt.onBlockUse(Quickreplant.kt:25)
	at net.fabricmc.fabric.api.event.player.UseBlockCallback.lambda$static$0(UseBlockCallback.java:41)
	at net.minecraft.class_636.handler$bka000$interactBlock(class_636.java:3681)
	at net.minecraft.class_636.method_2896(class_636.java:327)
	at net.minecraft.class_310.method_1583(class_310.java:1711)
	at net.minecraft.class_310.method_1508(class_310.java:1964)

-- Affected level --
Details:
	All players: 1 total; [class_746['TalEdd'/10145, l='ClientLevel', x=-860.32, y=64.00, z=-1088.88]]
	Chunk stats: 4096, 1411
	Level dimension: minecraft:overworld
	Level spawn location: World: (141,63,416), Section: (at 13,15,0 in 8,3,26; chunk contains blocks 128,-64,416 to 143,319,431), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 20974 game time, 20974 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:453)
	at net.minecraft.class_310.method_1587(class_310.java:2288)
	at net.minecraft.class_310.method_1514(class_310.java:758)
	at fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:37)
	at net.minecraft.class_310.modify$doe000$atTheEndOfSecondCatchBeforePrintingCrashReport(class_310.java:17939)
	at net.minecraft.class_310.method_1514(class_310.java:761)
	at fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:37)
	at net.minecraft.class_310.modify$doe000$atTheEndOfSecondCatchBeforePrintingCrashReport(class_310.java:17939)
	at net.minecraft.class_310.method_1514(class_310.java:761)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
```

**Comment by LordMZTE** [comment]
Thanks for the report! I totally forgot about the mangrove stuff. I'll look into this soon.

**Comment by LordMZTE** [comment]
Fixed in version `1.1.1`

**Comment by TalEdd** [comment]
Thank you!
